ansible/README.md

```diff
@@ -0,0 +1,168 @@
+# Ansible scripts for the Node.js build group infrastructure
+
+(in lack of a better title)
+
+
+## Getting started
+
+1. Install Ansible 2.2.0 or newer: `pip install ansible`. **Note**: if you're
+   using homebrew you'll have to manage dependencies such as `PyAML` yourself.
+2. Read this document.
+3. Clone the node secrets repository (if you don't have access ask anyone
+   in the [build group](https://github.com/nodejs/build#people).
+4. Copy the private keys (check the secrets repo for instructions) to your
+   `~/.ssh` folder. Make sure they have the same name. What keys are available
+   to you depends on what role you have. In order to create new vm's and hook
+   them up to CI you have to be part of the `infra` group.
+
+
+## Getting things done
+
+Most of your work will probably include editing `inventory.yml`, followed by
+running one (or multiple) of below playbooks. If you're adding a new host,
+limiting ansible to just running on that host is probably quicker:
+
+```console
+$ ansible-playbook playbooks/create-jenkins-worker.yml \
+    --limit "test-digitalocean-debian8-x64-1"
```

Comment:
I kind of assumed $workdir would be the same as the README.md. I guess we could add that somewhere.